Source/Core/Core/PowerPC/JitArm64/JitArm64_FloatingPoint.cpp

```diff
@@ -350,53 +350,42 @@ void JitArm64::fcmpx(UGeckoInstruction inst)
 	ARM64Reg VA = fpr.R(a, REG_IS_LOADED);
 	ARM64Reg VB = fpr.R(b, REG_IS_LOADED);
 
-	ARM64Reg WA = gpr.GetReg();
-	ARM64Reg XA = EncodeRegTo64(WA);
-
-	FixupBranch pNaN1, pNaN2, pNaN3, pLesser, pGreater;
+	FixupBranch pLesser, pGreater;
 	FixupBranch continue1, continue2, continue3;
-	ORR(XA, ZR, 32, 0, true);
 
 	if (a != b)
 	{
 		m_float_emit.FCMP(EncodeRegToDouble(VA), EncodeRegToDouble(VA));
 
 		// if (B != B) or (A != A), goto NaN's jump target
-		pNaN1 = B(CC_NEQ);
+		FALLBACK_IF(CC_NEQ);
```

Comment:
Please lookup the fallback macro, it's a compile-time boolean expression for calling the interpreter. So it must be done *before* anything else is emited, and it may not depend on any running data :/